astro/src/content/docs/apis/_report.mdx

```diff
@@ -35,10 +37,25 @@ This report includes the number of {props.reportType} for a given application or
     The start of the query range. This is an [instant](/docs/reference/data-types#instants) but it is truncated to {props.reportUnit} in the report timezone (which is set in the system settings).
   </APIField>
   <APIField name="loginId" type="String" optional since="1.4.0" renderif={props.reportURI === 'login'}>
-    When this parameter is provided it will reduce the scope of the report to a single user with the requested email or username specified by this parameter.
+    When this parameter is provided it will reduce the scope of the report to a single user. This parameter can be one of the following (see <InlineField>loginIdTypes</InlineField> for information on which of these identifiers will be used):
+
+    * Email address
+    * Phone number (if `phoneNumber` is included in <InlineField>loginIdTypes</InlineField>)
+    * Username
 
     This parameter is mutually exclusive with `userId`, if both are provided, the `loginId` will take precedence.
   </APIField>
+  <APIField name="loginIdTypes" type="Array<String>" optional defaults="email, username if email not found" since="1.99.9" renderif={props.reportURI === 'login'}>
```

Comment:
Since there are already 3 different URLs listed for this with different query param permutations, decided a fourth might be a bit much.